evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -850,7 +850,7 @@ def test_jupyter_graph_check(
     jupyter_check.return_value = True
     with pytest.warns(None) as graph_valid:
         graph_partial_dependence(clf, X, features=0, grid_resolution=20)
-        assert len(graph_valid) == 1
+        assert len(graph_valid) == 0
```

Comment:
Is there a particular reason that `graph_partial_dependence` was set to 1 here while the other graphing functions were 0?